tests/integration_tests/security/test_vulnerabilities.py

```diff
@@ -11,55 +11,140 @@
 import pytest
 import requests
 
-from framework import utils
-from framework.ab_test import git_ab_test_command_if_pr, set_did_not_grow_comparator
+from framework.ab_test import (
+    git_ab_test_command_if_pr,
+    git_ab_test_ssh_command,
+    git_ab_test_ssh_command_if_pr,
+    is_pr,
+    set_did_not_grow_comparator,
+)
 from framework.properties import global_props
 from framework.utils import CommandReturn
 from framework.utils_cpu_templates import nonci_on_arm
 
 CHECKER_URL = "https://meltdown.ovh"
 CHECKER_FILENAME = "spectre-meltdown-checker.sh"
+REMOTE_CHECKER_PATH = f"/tmp/{CHECKER_FILENAME}"
+REMOTE_CHECKER_COMMAND = (
+    f"sh {REMOTE_CHECKER_PATH} --explain --no-intel-db --batch json"
```

Comment:
Similar to "test: A/B-ify spectre_meltdown_checker_on_host", having both --explain and --batch json does not make sense.